servicex/transformer_manager.py

```diff
@@ -108,8 +114,11 @@ def create_job_object(request_id, image, chunk_size, rabbitmq_uri, workers,
                 client.V1EnvVar(name='MINIO_ACCESS_KEY',
                                 value=current_app.config['MINIO_ACCESS_KEY']),
                 client.V1EnvVar(name='MINIO_SECRET_KEY',
-                                value=current_app.config['MINIO_SECRET_KEY']),
+                                value=str(current_app.config['MINIO_SECRET_KEY'])),
```

Comment:
What type can the secret key if not str already?

Reply:
Yeah, that's a stray change from when I was debugging an issue.  I've fixed that.